### PR TITLE
Niloofar / Bug - Fix Typescript errors in landing/ebooks

### PR DESCRIPTION
**Pull request by niloo-fs**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by eduard-deriv — APPROVED** [review]

**Comment by eduard-deriv** [comment]
Cool! 👍

File: src/common/login.ts [hunk 1]
```diff
@@ -34,9 +36,10 @@ const Login = (() => {
             : `${oauth_url}/oauth2/authorize?app_id=${deriv_app_id}&l=${language}&brand=${brand_name.toLowerCase()}${affiliate_token_link}${cookies_link}&platform=${sub_url}`
     }
 
-    type TSocialProvider = 'google' | 'facebook' | 'apple'
-    const initOneAll = (provider: TSocialProvider): void => {
-        const social_login_url = `${loginUrl()}&social_signup=${provider}`
+    const initOneAll = (provider: TSocialProvider, utm_content?: string): void => {
```
Comment (ali-hosseini-fs) [inline src/common/login.ts:39]: in this section we didn't have `utm_content`.
where does that come from?

File: src/pages/landing/ebooks/components/_get-ebook.tsx [hunk 2]
```diff
@@ -147,17 +147,17 @@ const Line = styled.div`
     background-color: ${(props) => (props.color ? props.color : 'var(--color-grey-6)')};
 `
 
-const StyledText = styled(Text)`
+const StyledText = styled(Text)<{ tabletFontSize?: string }>`
     color: ${(props) => (props.color ? props.color : 'var(--color-grey-6)')};
 
-    @media ${(props) => device.tabletL && props.notedBox} {
+    @media ${device.tabletL} {
```
Comment (ali-hosseini-fs) [inline src/pages/landing/ebooks/components/_get-ebook.tsx:153]: dont we need this `notedBox` prop?

File: src/common/login.ts (hunk 1 above)
Comment (niloo-fs) [reply src/common/login.ts:39]: It comes from `_get-book` component.

File: src/pages/landing/ebooks/components/_get-ebook.tsx (hunk 2 above)
Comment (ali-hosseini-fs) [reply src/pages/landing/ebooks/components/_get-ebook.tsx:153]: so in that case this query never sets on styles, isn't it?

File: src/common/login.ts (hunk 1 above)
Comment (niloo-fs) [reply src/common/login.ts:39]: We are using this function inside `_get-book` component!

File: src/pages/landing/ebooks/components/_get-ebook.tsx (hunk 2 above)
Comment (niloo-fs) [reply src/pages/landing/ebooks/components/_get-ebook.tsx]: no we don't use this

**Review by iman-fs — APPROVED** [review]

**Pull request merged**